spec/compiler/codegen/array_literal_spec.cr

```diff
@@ -130,4 +130,50 @@ describe "Code gen: array literal spec" do
       custom.value
       )).to_i.should eq(6)
   end
+
+  it "creates custom non-generic array in module" do
+    run(%(
+      module Moo
+        class Custom
+          def initialize
+            @value = 0
+          end
+
+          def <<(element)
+            @value += element
+          end
+
+          def value
+            @value
+          end
+        end
+      end
+
+      custom = Moo::Custom {1, 2, 3}
```

Comment:
Nitpick: the formatter formats this as `custom = Moo::Custom{1, 2, 3}` so I think it should be the same in here.

Reply:
Yeah... but it was like that before, and this is just code inside text. I don't feel like looking around for all occurrences of this right now.

Reply:
Which is why I approved this PR :)